facia/app/controllers/FaciaController.scala

```diff
@@ -175,7 +176,11 @@ trait FaciaController
     } else result
 
   import PressedPage.pressedPageFormat
-  private[controllers] def renderFrontPressResult(path: String)(implicit request: RequestHeader) = {
+  private[controllers] def renderFrontPressResult(path: String)(implicit request: RequestHeader): Future[Result] = {
```

Comment:
I don't understand Scala very well so I just wanted to check what impact `: Future[Result]` has here, is this just a type check?

Reply:
Yes this is the function's expected return type. It is necessary as we are explicitly using a `return` statement which means this won't compile without an explicit return type

Reply:
Okay, thanks for clarifying 👍